implementation/test/DepositUtilsTest.js

```diff
@@ -376,7 +382,61 @@ describe("DepositUtils", async function() {
   })
 
   describe("auctionValue()", async () => {
-    it.skip("is TODO")
+    let duration
+    let basePercentage
+    before(async () => {
+      duration = await tbtcConstants.getAuctionDuration.call()
+      basePercentage = await tbtcConstants.getAuctionBasePercentage.call()
+      auctionValue = new BN(100000000)
+    })
+    beforeEach(async () => {
+      await createSnapshot()
+    })
+
+    afterEach(async () => {
+      await restoreSnapshot()
+    })
+
+    it("returns base value if no time has elapsed", async () => {
+      testDeposit.send(auctionValue, {from: accounts[0]})
+      const block = await web3.eth.getBlock("latest")
+
+      await testDeposit.setLiquidationAndCourtesyInitated(block.timestamp, 0)
+
+      const value = await testDeposit.auctionValue.call()
+      expect(value).to.eq.BN(auctionValue.mul(basePercentage).div(new BN(100)))
+    })
+
+    it("returns full value if auction Duration has elapsed ", async () => {
+      testDeposit.send(auctionValue, {from: accounts[0]})
+      const block = await web3.eth.getBlock("latest")
+
+      await testDeposit.setLiquidationAndCourtesyInitated(block.timestamp, 0)
+      await increaseTime(duration.toNumber())
```

Comment:
Is this potentially subject to the issue we just fixed in #465 ? Should we add some extra factor here to avoid intermittent failure?

Reply:
yes, potentially. We could test margin here too